Foundation/NSFileManager.swift

```diff
@@ -343,7 +343,19 @@ open class FileManager : NSObject {
         This method replaces fileSystemAttributesAtPath:.
      */
     open func attributesOfFileSystem(forPath path: String) throws -> [FileAttributeKey : Any] {
-        NSUnimplemented()
+        var s = statfs()
+        guard statfs(path, &s) == 0 else {
```

Comment:
@phausler I mentioned this issue in conversation, however I considered `attributesOfItem()` method implementation to address. If we should correct something here, that will also effect `attributesOfItem()` method too.